src/amqp-ts.ts

```diff
@@ -874,12 +868,11 @@ export class Queue {
         });
       }).catch((error) => {
         log.debug( "Queue publish error: " + error.message, { module: "amqp-ts", error });
-        var queueName = this._name;
         var connection = this._connection;
         log.debug("Try to rebuild connection, before Call.", { module: "amqp-ts" });
         return connection._rebuildAll(error).then(() => {
-          log.debug("Retransmitting message.", { module: "amqp-ts" });
-          return connection._queues[queueName].publish(content, options);
+          log.warn("Connection rebuilt, NOT retransmitting message.", { module: "amqp-ts" });
```

Comment:
Same comment as the other one. I didn't check which method we are currently using but we should ideally just keep one and remove the rest.

Reply:
When discussing with @dygcao, it was decided to avoid removing deprecated methods to avoid including too many changes in the PR. @dygcao Do you still have that opinion?

Reply:
Yes, for amqp-ts, I prefer that we only change what's necessary. I believe publish isn't in use, but since it's hard to be sure, I'd rather keep the changes separate in case we decide to clean up deprecated methods. However, in the long term, I don't think we plan to maintain amqp-ts, so it's probably not worth the effort.

Reply:
I'm okay to do this separately in another PR.